app/mailers/friends_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class FriendsMailer < ActionMailer::Base
+  layout 'mailers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.